src/routes/reference/basic-reactivity/create-signal.mdx

```diff
@@ -39,7 +39,7 @@ Calling the getter (e.g., `count()` or `ready()`) returns the current value of t
 Crucial to automatic dependency tracking, calling the getter within a tracking scope causes the calling function to depend on this Signal, so that function will rerun if the Signal gets updated.
 
 Calling the setter (e.g., `setCount(nextCount)` or `setReady(nextReady)`) sets the Signal's value and updates the Signal (triggering dependents to rerun) if the value actually changed (see details below). 
-The setter takes either the new value for the signal or a function that maps the last value of the signal to a new value as its only argument. 
+The setter takes either the new value for the signal or a function that maps the latest value of the signal to a new value as its only argument. 
```

Comment:
I think `last` (or `previous`) is more correct here, as we're about to set a new next value.